src/collections/_documentation/workflow/alerts-notifications/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Alerts & Notifications
+sidebar_order: 0
+---
+
+{% capture __alert_content -%}
+Sentry's Performance features are currently in beta. For more details about access to these features, feel free to reach out at [https://sentry.io/for/performance/](https://sentry.io/for/performance/).
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Sentry notifications are broadly categorized into alerts and non-alert notifications. Alerts can be sent to many supported [LINK: integrations]. Non-alert notifications only go to email.
+
+## Alerts
+
+Alerts are configured per-project and are of two types: Metric Alerts and Issue Alerts.
+
+### Metric Alerts
+
+Metric alerts allow you to filter and set thresholds across all errors in a project. They can be used for high-level monitoring of patterns, or fine-grained monitoring of individual events. Metric alerts help express lightweight service-level objectives (SLOs) such as users affected by signup-page errors, or volume of database errors. For more details, see the [LINK: full documentation on Metric Alerts].
+
+### Issue Alerts
+
+Issue alerts fire whenever any issue in the project matches the specified criteria, such as a resolved issue re-appearing or an issue affecting many users.
+
+For more details, see the [LINK: full documentation on Issue Alerts].
+
+[ screenshot of Alerts in-app ]
+
+Some alert settings apply at the project level (for example, to all alerts in the project), while others are specific to the alert. You can also tweak your personal alerts settings to control which alerts you receive.
```

Comment:
can we replace "for example" with "that is" here? it's not an example, it _means_ "to all alerts in the project"